src/Security/Security.Test/Security.Test.csproj

```diff
@@ -10,38 +10,12 @@
     <RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName)$(AzTestAssemblySuffix)</RootNamespace>
   </PropertyGroup>
 
+  <!--<ItemGroup>
+    <None Update="PemFiles\*.*" CopyToOutputDirectory="PreserveNewest" />
+  </ItemGroup>-->
+
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.SecurityCenter" Version="2.1.0" />
-  </ItemGroup>
-  
-  <ItemGroup>
```

Comment:
Is this removal on purpose? why was it added to begin with?

Reply:
IT should not been added.
The Test.csproj should update the nuget to the latest version